tools/reviewer/job/sync/full_github_pr.proto

```diff
@@ -30,30 +32,12 @@ message CommitPointer {
   User user = 3;
 }
 
-message Review {
-  enum State {
-    UNKNOWN = 0;
-    APPROVED = 1;
-    CHANGES_REQUESTED = 2;
-    COMMENTED = 3;
-    PENDING = 4;
+message ReviewComment {
+  message Links {
+    message PullRequestLink { string href = 1; }
```

Comment:
**oferb's comment:**
What is `href`? If it's a url, perhaps rename to `url`?

Reply:
Yes, it is `URL`. E.g. `https://api.github.com/repos/octocat/Hello-World/pulls/1`
It has already renamed to url